### DISCUSSION TITLE
Should we use thumbnails on the left or under the main image on the product page?

### DISCUSSION THREAD

**Discussion by NeOMakinG** [PrestaShop/hummingbird#410] [Polls]
I would like to talk about the thumbnails on the left on the product page

I agree on every big market places uses thumbnails on the left, but I would love to avoid it in the base theme:

- It forces us to add a new library in order to have a slider while we could let users add it themselves
- It's hardening the maintenance of the theme
- A lot of contributors there don't want it while no one is still asking for it atm

**Comment by SharakPL** [comment]
It's easier to make it under + on mobile it should be under anyway, because there's not enough space on the left.

**Comment by dennispw** [comment]
From our point of view: It doesn't really matter. Most of *our* clients don't really care that much. As long as the UX is good, it doesn't really matter *to them* if they are horizontal or vertical. However, the clients that *really* want vertical thumbnails are quite fierce about it, and are usually already using a third party theme or are in the middle of developing one. Most of the studies I've skimmed through suggests that vertical thumbnails is a bad pattern anyhow. As such we believe this should really be up to the theme developers to decide and integrate. 

Personally I would advise against using any particular script/library/framework for this, whether it's horizontal *or* vertical. Make it *very* basic to begin with.  
Horizontal thumbnails? `overflow-x: auto`.  
Vertical thumbnails? `overflow-y: auto`.  
Of course there is more to it than that but to get the point across.

In my mind the standard/default theme should *not* try do do everything but rather do what it actually does really well and efficiently.

If a script/library/framework *is* used, please make sure that it is removable and/or highly customizable, don't tie anything more into the core (looking at you jQuery UI!).

**Reply by NeOMakinG** [reply]
Interesting feedback, thanks :)

**Comment by matks** [comment]
I think we can close this topic after a year 😄
